### PR TITLE
report PR number to benchmark collector

### PR DESCRIPTION
**Pull request by kadel**
Adds PR number into the [benchmark results table](https://docs.google.com/spreadsheets/d/1o-GIoYlZoEyW1F25kAwvvEXzbW4tLck0x3EGCC4_L_A/edit#gid=0)

### CODE REVIEW

File: tests/helper/reporter/httpMeasurementReporter.go [hunk 1]
```diff
@@ -36,6 +37,7 @@ func (r *HTTPMeasurementReporter) SpecDidComplete(specSummary *types.SpecSummary
 			output := map[string]string{}
 			output["Number of Samples"] = strconv.Itoa(specSummary.NumberOfSamples)
 			output["Measurement"] = k
+			output["PR"] = os.Getenv("PULL_NUMBER")
```
Comment (mik-dass) [inline tests/helper/reporter/httpMeasurementReporter.go]: a comment here regarding this env variable

Comment (kadel) [reply tests/helper/reporter/httpMeasurementReporter.go]: updated

**Comment by mik-dass** [comment]
```
[odo]  ✗  waited 4m0s but couldn't find running pod matching selector: 'deploymentconfig=mycomponent-myapp'
```
/retest

**Comment by girishramnani** [comment]
Two approves
/lgtm

**Pull request merged**